From: Rene Kita <mail@rkta.de>
Date: Mon May 20 09:52:00 CEST 2024
Subject: Add option to use same cookie file format as curl
Patch: patches/Add-option-to-use-same-cookie-file-format-as-curl.patch

---
 cookie.c |   13 +++++++++----
 1 file changed, 9 insertions(+), 4 deletions(-)

--- a/cookie.c
+++ b/cookie.c
@@ -530,10 +530,15 @@ save_cookies(void)
 	if (!(p->flag & COO_USE) || p->flag & COO_DISCARD)
 	    continue;
 	fprintf(fp, "%s\t%s\t%s\t%ld\t%s\t%s\t%d\t%d\t%s\t%s\t%s\n",
-		parsedURL2Str(&p->url)->ptr,
-		p->name->ptr, p->value->ptr, (long)p->expires,
-		p->domain->ptr, p->path->ptr, p->flag,
-		p->version, str2charp(p->comment),
+		parsedURL2Str(&p->url)->ptr,	/* url */
+		p->name->ptr,
+		p->value->ptr,
+		(long)p->expires,
+		p->domain->ptr,
+		p->path->ptr,
+		p->flag,
+		p->version,
+		str2charp(p->comment),
 		(p->portl) ? portlist2str(p->portl)->ptr : "",
 		str2charp(p->commentURL));
     }
